SAKPaaS/src/app/components/occupancy-report/occupancy-report.component.ts

```diff
@@ -54,34 +56,35 @@ export class OccupancyReportComponent implements OnInit, OnDestroy {
         messageKey: 'snack-bar.occupancy-report.invalid',
         type: SnackBarTypes.ERROR
       });
-      return;
+      return undefined;
     }
     const value = +this.occupancyInput.value;
 
     this.isLoadingService.add({ key: 'sendOccupancy' });
 
-    this.subscriptions.add(
-      this.selectedLocation$.pipe(
-        switchMap(location => {
-          return this.occupancyService.sendOccupancy(location.id, value);
-        }),
-        catchError(err => {
-          this.isLoadingService.remove({ key: 'sendOccupancy' });
-          this.snackBarService.sendNotification({
-            messageKey: 'snack-bar.occupancy-report.failure',
-            type: SnackBarTypes.ERROR
-          });
-          return throwError(err);
-        })
-      ).subscribe(location => {
+    this.selectedLocation$.pipe(
```

Comment:
Warum entfernst du hier das `this.subscriptions.add(…)`?

Reply:
Good question.

Reply:
Fixed.